### PR TITLE
add package: php-psr

### PR DESCRIPTION
**Pull request by ian4hu**
New package: php-psr

php-psr: PHP extension providing the accepted PSR interfaces

### CODE REVIEW

File: packages/php-psr/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/jbboehr/php-psr
+TERMUX_PKG_DESCRIPTION="PHP extension providing the accepted PSR interfaces"
+TERMUX_PKG_LICENSE="Simplified BSD"
```
Comment (thunder-coding) [inline packages/php-psr/build.sh:3]: `BSD Simplified` and not `Simplified BSD`

File: packages/php-psr/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_LICENSE_FILE=LICENSE.md
```
Comment (thunder-coding) [inline packages/php-psr/build.sh:4]: Not required

**Review by thunder-coding — CHANGES_REQUESTED** [review]

File: packages/php-psr/build.sh (hunk 2 above)
Comment (ian4hu) [reply packages/php-psr/build.sh:4]: `TERMUX_PKG_LICENSE_FILE ` is required, without it the build scripts return an `ERROR: No LICENSE file was installed for php-psr`

File: packages/php-psr/build.sh (hunk 1 above)
Comment (ian4hu) [reply packages/php-psr/build.sh]: Just corrected it.

Comment (thunder-coding) [reply packages/php-psr/build.sh]: The package seems to be `BSD Simplified` and not `BSD`.

`BSD Simplified` license is available

File: packages/php-psr/build.sh (hunk 2 above)
Comment (Grimler91) [reply packages/php-psr/build.sh:4]: It should not be required, LICENSE.md is in the list of searched file names: https://github.com/termux/termux-packages/blob/5028aaca6ad098bba95775d64a7ccd294ed9412a/scripts/build/termux_step_install_license.sh#L38

Comment (Grimler91) [reply packages/php-psr/build.sh:4]: My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a, so if you rebase this onto master you can drop that line

Comment (ian4hu) [reply packages/php-psr/build.sh]: I test it again, the error is not occur, and I removed the `TERMUX_PKG_LICENSE_FILE`

**Review by thunder-coding — APPROVED** [review]

File: packages/php-psr/build.sh (hunk 2 above)
Comment (ian4hu) [reply packages/php-psr/build.sh]: > My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in [73bc3b1](https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a), so if you rebase this onto master you can drop that line

done

**Review by thunder-coding — APPROVED** [review]

File: packages/php-psr/build.sh (hunk 2 above)
Comment (ian4hu) [reply packages/php-psr/build.sh]: > My bad, Simplified BSD is in the list of licenses with copyright information. I fixed that in [73bc3b1](https://github.com/termux/termux-packages/commit/73bc3b19ca92023ca1244f8db83d7699c5395d2a), so if you rebase this onto master you can drop that line

As @thunder-coding said: it should be `BSD Simplified` , not `Simplified BSD`

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**